src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -10,6 +10,7 @@ unreleased
 * vm: fix "vm extension list" crash if the VM has no extensions
 * vmss: update arg description for 'vmss delete-instances --instance-ids'
 * vmss: hide arg 'vmss show --ids', which is not supposed to work because of 'instance-id' arg
+* vmss: including instance ids in list_vmss_instance_connectio_info
```

Comment:
Recommend: "BC: `vmss list-instance-connection-info`: Include instance IDs in the output."  Users care about the command, not the internal method. Also, it's handy to call out that this is a breaking change in the release notes so users can update their script if needed (hence the BC)